src/hotspot/cpu/x86/x86.ad

```diff
@@ -3247,10 +3249,11 @@ instruct sqrtF_reg(regF dst, regF src) %{
 instruct sqrtF_mem(regF dst, memory src) %{
   predicate(UseSSE>=1);
   match(Set dst (SqrtF (LoadF src)));
-
+  effect(TEMP dst);
```

Comment:
Why do you declare `dst` as `TEMP` (here and in other places)?

Reply:
Good point. I was being extra cautious that the register in address should not be overwritten by the xorps. But of course the address cannot have xmm register here so TEMP dst is not needed.  I will update the patch accordingly.